WaterSkiBaan/SportOpslag/SkiOpslag.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WaterSkiBaan.SportUitrusting;
+
+namespace WaterSkiBaan.SportOpslag
+{
+    public class SkiOpslag : IOpslag
+    {
+        private Stack<Skies> _opslag { get; set; } = new Stack<Skies>();
+
+        public void Afgeven(SportArtikel sportartikel)
+        {
+            var skies = (Skies)sportartikel;
+            _opslag.Push(skies);
+        }
+
+        public Skies PakSkies()
+        {
+            return _opslag.Pop();
+        }
+
+        public override string ToString()
+        {
+            var opslagArray = _opslag.ToArray();
+            StringBuilder sb = new StringBuilder();
+            for (int i = 0; i < opslagArray.Length; i++)
+            {
+                sb.Append("* ");
+            }
+            return $"({opslagArray.Length.ToString()}) {sb.ToString()}";
```

Comment:
Redundant 'Object.ToString()' call

Reply:
Hmjaja, ik weet niet wat dit precies is. Ik heb dit niet geschreven, dit hoort bij het waterskibaalskelet.

Reply:
Hmjaja, ik weet niet wat dit precies is. Ik heb dit niet geschreven, dit hoort bij het waterskibaalskelet.